### DISCUSSION TITLE
Exception thrown when adding a profile name

### DISCUSSION THREAD

**Discussion by JDtroles** [AsamK/signal-cli#456] [Q&A]
I tried to update the profile with a name, but it threw the following exception:

> signal-cli -u +49xyz updateProfile --name "Test"
> WARN App - WARNING: Support for new group V2 is disabled, because the required native library dependency is missing: libzkgroup
> Exception in thread "main" java.lang.NoClassDefFoundError: Could not initialize class org.signal.zkgroup.internal.Native
>         at org.signal.zkgroup.profiles.ProfileKey.getProfileKeyVersion(ProfileKey.java:47)
>         at org.whispersystems.signalservice.api.SignalServiceAccountManager.setVersionedProfile(SignalServiceAccountManager.java:730)
>         at org.asamk.signal.manager.Manager.setProfile(Manager.java:358)
>         at org.asamk.signal.commands.UpdateProfileCommand.handleCommand(UpdateProfileCommand.java:37)
>         at org.asamk.signal.App.handleLocalCommand(App.java:209)
>         at org.asamk.signal.App.init(App.java:165)
>         at org.asamk.signal.Main.main(Main.java:43)

Same happens if I drop the quotes before and after the name.

Is there a way to fix this from my side?

Thanks for the help

**Comment by AsamK** [comment]
Setting a profile also requires the native libzkgroup: https://github.com/AsamK/signal-cli/wiki/Provide-native-lib-for-libsignal

**Reply by JDtroles** [reply]
Thank you for the fast answer! Great support :)
